plugins/modules/namespace_info.py

```diff
@@ -63,7 +67,7 @@ def main():
     path = '/namespaces'
 
     try:
-        namespaces = utils.get(client, path)
+        namespaces = utils.prepare_result_list(utils.get(client, path))
```

Comment:
Actually, this is an old remnant that I did not intend to commit. We cannot fetch a single namespace, which means that we do not need to filter the result of the `GET` call.

I know that having this here does not hurt us, but I would rather keep it out because this allows our integration tests to catch incompatible changes in the Sensu's web API.